Makefile

```diff
@@ -125,12 +125,16 @@ test-python:
 test-python-ci:
 	make build-platform-assets
 	@echo "--> Running CI Python tests"
-	pytest tests/integration tests/sentry --cov . --cov-report="xml:.artifacts/python.coverage.xml" --junit-xml=".artifacts/python.junit.xml" || exit 1
+	pytest tests/integration tests/sentry \
+		--ignore tests/sentry/eventstream/kafka \
```

Comment:
backend tests do run snuba, but we're already running the same tests in snuba tests.